### ISSUE TITLE
Alert banners convey meaning using colour alone by types having styling reflecting their purpose

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_alert_banner#345]
For example, _Death of a notable person_ uses a black background and white text, _Major_ a red background.

Setting a role and an accessible name (the alert banner type) on one the containing elements should fix.
